tests/integration/nmstatectl_test.py

```diff
@@ -118,8 +118,12 @@ def test_set_command_with_yaml_format():
     ret = libcmd.exec_cmd(SET_CMD, stdin=ETH1_YAML_CONFIG)
     rc, out, err = ret
 
-    assert_rc(rc, RC_SUCCESS, ret)
+    assert_rc(rc, RC_SUCCESS, exec_cmd_report(ret))
 
 
-def assert_rc(actual, expected, return_tuple):
-    assert actual == expected, 'rc={}, out={}, err={}'.format(*return_tuple)
+def assert_rc(actual, expected, assert_report):
```

Comment:
IMHO it would be better to make `assert_rc` less universal and more specific for the current use case, i.e. only pass `expected` and `return_tuple` into it. Currently it is just harder to use and does not follow the KISS principle.

Reply:
Dropping it now make sense, as you suggested in the previous comment.